src/main/java/tracing/SNT.java

```diff
@@ -0,0 +1,26 @@
+package tracing;
+
+import org.scijava.util.VersionUtils;
+
+import ij.IJ;
+
+/** Static utilities for SNT **/
+public class SNT {
+	public static final String VERSION = getVersion();
+
+	private SNT() {
+	}
+
+	private static String getVersion() {
+		return VersionUtils.getVersion(tracing.SimpleNeuriteTracer.class);
+	}
+
+	protected static void error(final String string) {
+		IJ.error("Simple Neurite Tracer v" + getVersion(), string);
+	}
+
+	protected static void log(final String string) {
+		System.out.println("[SNT] " + string);
```

Comment:
@tferr Actually, I think that looks good. It's precisely how you have to access the SciJava context from IJ1-land. There is no need for the `@Parameter` annotations though; they don't really do anything here. The only other consideration is whether the SNT class needs to be thread-safe. If not, what you have will work. But if you need thread safety, you can use a double-checked locking pattern like many of the SciJava services do for their lazy initialization. (You can do `git grep 'lazy init'` to find lots of examples in the scijava-common codebase.)